### ISSUE TITLE
Keypoints module not found

### ISSUE THREAD

**Issue by hungxiu1995** [davidcaron/pclpy#23]
I can't find keypoints module in pip-installed pclpy. Does compiling from source help?

**Comment by davidcaron** [comment]
You're right, it's missing... it's not a module I'm familiar with, so I would need to investigate more.
A couple files are explicitly skipped because I had some issues when compiling, but I didn't realize that the whole module was absent. I'm not sure why at the moment.

**Comment by AntoineGuiot** [comment]
Hi @davidcaron, thank you for your great work! 

Do you have any update about this issue?
